dds/DCPS/transport/framework/TransportRegistry.cpp

```diff
@@ -150,6 +149,13 @@ TransportRegistry::load_transport_configuration(const OPENDDS_STRING& file_name,
                                 transport_id.c_str()),
                                -1);
             }
+
+            // store the transport info
+            TransportEntry entry;
+            entry.transport_name = transport_id;
+            entry.transport_info = values;
+            transports_.push_back(entry);
```

Comment:
Would any of the failure cases below require us to purge / skip this transport, or are they all severe enough it won't matter that we're modifying transports_ here?